budoux/main.py

```diff
@@ -34,21 +35,53 @@ class BudouxHelpFormatter(argparse.ArgumentDefaultsHelpFormatter,
 
 
 def check_file(path: str) -> str:
-  """Check if filepath is exist or not.
+  """Check if a given filepath exists or not.
 
   Args:
       path (str): Model path
 
   Raises:
-      FileNotFoundError: Raise if given path is not exist.
+      FileNotFoundError: Raise if given path does not exist.
 
   Returns:
-      str: Model path confirmed its existance.
+      str: A model path.
   """
   if os.path.isfile(path):
     return path
   else:
-    raise FileNotFoundError("'{}' is not found.".format(path))
+    raise argparse.ArgumentTypeError(f"'{path}' is not found.")
+
+
+def get_model_langs() -> typing.Dict[str, str]:
+  """Get a dictionary of model languages and its paths.
+
+  Returns:
+      typing.Dict[str, str]: A dictionary of model languages and its paths.
+  """
+  models = glob.glob(
+      pkg_resources.resource_filename(__name__, "models") + "/*-*.json")
```

Comment:
We may want to change this function to return `ja` and `zh-hans`, too.
Sorry I understand that the new model file is breaking the naming convention of `[lang-code]-[data-source].json` :(

Reply:
1c03f6e